### PR TITLE
Format numbers on metrics page (react)

### PR DESCRIPTION
**Pull request by qmonmert**
React : 

Format the numbers on the metrics page (same display that angular app).

### CODE REVIEW

**Comment by deepu105** [comment]
I guess instead of adding libs here for each format, I'm thinking of making a wrapper in the react-jhipster project which does these and then uses them like angular pipe here. I'll make the required changes in the lib and let you know so that you can amend this PR to use those

**Comment by qmonmert** [comment]
ok with that

**Comment by deepu105** [comment]
@qmonmert please use `TextFormat` component from react-jhipster for this. Similar to https://github.com/jhipster/generator-jhipster/commit/f0689314220ffceb81079043bc0711cd7a6f60eb

I guess to have a format like `10,000` the below would be enough, to specify any other format use this syntax http://numeraljs.com/#format
```
<TextFormat value={number} type="number" />
```

**Comment by qmonmert** [comment]
@deepu105 I use TextFormat now ;)

File: generators/client/templates/react/src/main/webapp/app/config/constants.ts [hunk 1]
```diff
@@ -9,3 +9,8 @@ export const messages = {
   DATA_DELETE_SUCCESS_ALERT: 'Data deleted successfully!',
   DATA_UPDATE_ERROR_ALERT: 'Data deleted successfully!'
 };
+
+export const APP_DATE_FORMAT = 'DD/MM/YY HH:mm';
+
+export const APP_FORMAT_NUMBER_WHOLE = '0,0';
```
Comment (deepu105) [inline generators/client/templates/react/src/main/webapp/app/config/constants.ts]: could you please change this to `APP_NUMBER_WHOLE_FORMAT` & `APP_NUMBER_TWO_DIGITS_AFTER_POINT_FORMAT` for consistency

Comment (deepu105) [reply generators/client/templates/react/src/main/webapp/app/config/constants.ts]: never mind ill change it

Comment (qmonmert) [reply generators/client/templates/react/src/main/webapp/app/config/constants.ts]: ok I was ready to change ;)

**Comment by deepu105** [comment]
@sendilkumarn I think you broke lint, can you fix it

**Pull request merged**